src/lib/util/randomVocab.ts

```diff
@@ -311,4 +322,27 @@ export class RandomVocabContext {
 		);
 		return {role, persona, account, community};
 	}
+
+	async policy(
+		community?: Community,
+		persona?: Persona,
```

Comment:
```suggestion
		persona?: AccountPersona,
```

I think these 2 changes fix CI